src/__test__/tweetAuthor.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import TweetAuthor from '../components/TweetAuthor'
+
+function setup() {
+  const props = {
+    showModal: jest.fn(),
+    user: 'user'
```

Comment:
This isn't going to work properly. User needs to be an object. You are testing for class names existing but not the correct data being rendered in the right place, which is a more valuable test.